sos/plugins/foreman.py

```diff
@@ -17,13 +19,136 @@ class Foreman(Plugin, RedHatPlugin):
 
     plugin_name = 'foreman'
     profiles = ('sysmgmt',)
-    packages = ('foreman-debug',)
+    packages = ('foreman', 'foreman-proxy')
+    option_list = [
+        ('months', 'number of months for dynflow output', 'fast', 6)
+    ]
 
     def setup(self):
-        cmd = "foreman-debug"
+        self.add_forbidden_path([
+            "/etc/foreman*/*key.pem",
+            "/etc/foreman*/encryption_key.rb"
+        ])
 
-        path = self.get_cmd_output_path(name="foreman-debug")
-        self.add_cmd_output("%s -g -q -a -d %s" % (cmd, path),
-                            chroot=self.tmp_in_sysroot(), timeout=900)
+        self.add_copy_spec([
+            "/etc/foreman/",
+            "/etc/foreman-proxy/",
+            "/etc/sysconfig/foreman",
+            "/etc/default/foreman",
+            "/etc/foreman-installer/",
+            "/var/log/foreman/dynflow_executor*.log*",
+            "/var/log/foreman/dynflow_executor*.output*",
+            "/var/log/foreman/apipie_cache*.log*",
+            "/var/log/foreman/cron*.log*",
+            "/var/log/foreman/db_migrate*.log*",
+            "/var/log/foreman/db_seed*.log*",
+            "/var/log/foreman/production.log*",
+            "/var/log/foreman-proxy/cron*.log*",
+            "/var/log/foreman-proxy/migrate_settings*.log*",
+            "/var/log/foreman-proxy/proxy*.log*",
+            "/var/log/foreman-proxy/smart_proxy_dynflow_core*.log*",
+            "/var/log/foreman-selinux-install.log",
+            "/var/log/foreman-proxy-certs-generate*",
+            "/var/log/foreman-installer/",
+            "/var/log/foreman-maintain/",
+            "/var/log/syslog*",
+            # Specific to Sat6, _all_ catalina logs are relevant. Adding this
+            # here rather than the tomcat plugin to ease maintenance and not
+            # pollute non-Sat sosreports that enable the tomcat plugin
+            "/var/log/tomcat*/catalina*.log*",
+            "/var/log/tomcat*/host-manager*.log*",
+            "/var/log/tomcat*/localhost*.log*",
+            "/var/log/tomcat*/manager*.log*",
+            "/usr/share/foreman/Gemfile.in"
+        ])
+
+        self.add_cmd_output([
+            'bundle --local --gemfile=/usr/share/foreman/Gemfile.in',
```

Comment:
The file name on Debian is just `Gemfile`, please call here:

    bundle --local --gemfile=/usr/share/foreman/Gemfile*